keepalived-vip/main.go

```diff
@@ -54,6 +50,8 @@ var (
 
 func main() {
 	clientConfig := kubectl_util.DefaultClientConfig(flags)
+
+	flags.AddGoFlagSet(flag.CommandLine)
```

Comment:
Ah, does that show up in --help? (how to increase verbosity i mean), if not might be worth adding it, or noting it in some doc.